dask_bigquery/core.py

```diff
@@ -89,6 +94,8 @@ def read_gbq(
     dataset_id: str,
     table_id: str,
     row_filter="",
+    *,
+    cred_fpath: str = None,
```

Comment:
A credentials object is going to be the most flexible. Not everyone has access to the local file system, such as if they're running from a Colab notebook.